contracts/GenArt721CoreV3.sol

```diff
@@ -162,30 +160,31 @@ contract GenArt721CoreV3 is ERC721Enumerable, IGenArt721CoreContractV3 {
         internal
         returns (uint256 _tokenId)
     {
-        uint256 tokenIdToBe = (_projectId * ONE_MILLION) +
+        uint256 nextTokenId = (_projectId * ONE_MILLION) +
             projects[_projectId].invocations;
 
         projects[_projectId].invocations = projects[_projectId].invocations + 1;
 
-        bytes32 hash = keccak256(
+        bytes32 tokenHash = keccak256(
             abi.encodePacked(
                 projects[_projectId].invocations,
                 block.number,
```

Comment:
`block.number` doesn't add any additional randomness (we are already including blockhash of prior block), so we should strip it out.

Reply:
True. Removing.